api/src/main/scala/io/phdata/rest/WorkspaceController.scala

```diff
@@ -361,17 +361,22 @@ class WorkspaceController[F[_]: Sync: Timer: ContextShift: ConcurrentEffect](
           if (user.isOpsUser) {
             for {
               workspace <- workspaceService.findById(id).value
-              _ <- hdfsService
-                .setQuota(
-                  workspace.get.data.find(hiveAllocation => hiveAllocation.id.get == resourceId).get.location,
-                  size,
-                  resourceId,
-                  Instant.now()
-                )
-                .onError {
-                  case e: Throwable =>
-                    logger.error(s"Failed to modify disk-quota of workspace $id: ${e.getLocalizedMessage}", e).pure[F]
-                }
+              hiveAllocation <- workspace.get.data.find(_.id.get == resourceId).get.pure[F]
+              _ <- if (hiveAllocation.isUsingHDFS) {
```

Comment:
We should store the scheme the user is actually using so it can be displayed on the ui.

Then instead of a predicate check you can pattern match

```
hive.dataSource match {

case "hdfs" =>
case _ => 
```

This way also we can show on the front end the scheme so users know this database is in s3 or HDFS or ADLS or whatever

Reply:
how about we call the new property `protocol`?